### DISCUSSION TITLE
Duda VUE

### DISCUSSION THREAD

**Discussion by GloriaCoralCerecedo** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#508] [Q&A]
Buenas noches

¿A que se refiere esto en la práctica de VUEX?

La información de sabores, adornos, cantidades y precio debe de estar en el estado con Vuex o Vite y se debe modificar ahí para mostrarse en los componentes.

**Comment by MauroMontan** [comment]
VUEX sirve para tener información global disponible para todos los componentes sin necesidad de usar props, puedes ver más en la documentación [vuex](https://vuex.vuejs.org/), o puedes usar tambien pinia, es una buena alternativa a VUEX, y en mi opinión es más facil de usar y mas comodo  [pinia](https://pinia.vuejs.org/),

para la practica puedes mantener toda la información de los pasteles dentro de VUEX para que este disponibles en todos los componentes

**Reply by GloriaCoralCerecedo** [reply]
Oh ya es como el ejemplo de title de "Hola explorers" son como variables que ya estan predefinidas y solo se mandan a llamar no¿?

**Reply by MauroMontan** [reply]
si, en el store creado con vuex puede colocar toda la información de los pasteles, y de sus usuarios

**Reply by GloriaCoralCerecedo** [reply]
Seria en el state

**Comment by Burstniart** [comment]
Con Vuex se refiera aque en vez de escribir tus variables dentro del html las consultes desde el state como el ejemplo del contador, y desde ahi seran modificadas tambien.
Entonces sabores, adornos, cantidades y precio deberan estar en el state

**Reply by GloriaCoralCerecedo** [reply]
Oh ya esas variables se ponen ahí para que no se repitan y de ahí se mandan a llamar

**Comment by ArmandoRuiz13** [comment]
Se que este no es una discusion propia, para evitar crear una, saben como se obtienen las imagenes de la carpeta assets en vue, no logro encontrar el src indicado

**Reply by ArielCanedo** [reply]
Hola bro
Igual que en html
En (src=“../assets/nombre-de-la-imagen.png” y guardas el archivo, ahí ya te debería aparecer

**Reply by ArmandoRuiz13** [reply]
Gracias, creo que tiene un error mi archivo no se  porque no funciona.

**Reply by raulquintero13** [reply]
si las tienes en la carpeta 'src' ponlas en la carpeta 'public',
